resources/views/components/user/profile/last-seen-in.blade.php

```diff
@@ -10,9 +10,9 @@
 ])
 
 <?php
-$mostRecentSession = PlayerSession::where('user_id', $userMassData['ID'])
+$mostRecentSession = GameRecentPlayer::where('user_id', $userMassData['ID'])
     ->with('game')
-    ->orderBy('created_at', 'desc')
+    ->orderByDesc('rich_presence_updated_at')
     ->first();
```

Comment:
`PruneGameRecentPlayers` will truncate the table daily. For someting like the AotW, this may cause user's entries to disappear before they play another game.

I was going to suggest falling back to the slower query if the first one fails, but there is likely going to be _some_ game where the user is still a recent player.